nx/lib/nx/constants.ex

```diff
@@ -292,6 +292,19 @@ defmodule Nx.Constants do
     Nx.tensor(Complex.new(0, 1), opts)
   end
 
+  @doc """
+  Returns the smallest normal floating point number.
+
+  The smallest normal floating point number is the smallest positive value with
+  1 as the leading bit for the mantissa.
+
+  Accepts the same options as `Nx.tensor/2`
+  """
+  def smallest_normal(type, opts \\ []) do
```

Comment:
this is the name Jax uses, but I'm not sure if we should use `min_normal` to keep with `min_finite`

Reply:
I think we had this in the past as `min_positive` or something like that and we removed it because it was no longer used.

Reply:
https://github.com/elixir-nx/nx/commit/fc9a2e7148a1d54e35fdd6edb38197c924d3e1d0

found it!

Shall I restore it instead?

Reply:
Whatever you prefer!